### PR TITLE
Fix local Dockerfile build for tito changes

### PR DESCRIPTION
**Pull request by mkutlak**
#### Issue 1:
tito crashed with `_curses.error: setupterm: could not find terminal`
during the Dockerfile build.

Specifying TERM env variable works around the issue.

bugzilla:
bugzilla.redhat.com/show_bug.cgi?id=1733043

#### Issue 2:
- Install tito for build
- Use dnf builddep to install dependencies
- Change paths to rpms

### CODE REVIEW

**Comment by ernestask** [comment]
Fwiw, I’m trying something completely different, but rpmbuild is mysteriously failing.

**Comment by ernestask** [comment]
But now it works after a kernel upgrade… Probably something with overlayfs.

File: container/Dockerfile_local [hunk 1]
```diff
@@ -11,9 +13,9 @@ WORKDIR '/faf'
 # Change owner of /faf, clean git and install dependences
 RUN chown -R --silent faf /faf && \
     chmod -R --silent g=u /faf && \
-    dnf -y install rpm-build sudo git which vim make &&  \
+    dnf -y install git make rpm-build sudo tito vim which && \
```
Comment (ernestask) [inline container/Dockerfile_local]:
```suggestion
    dnf -y install git-core make rpm-build sudo tito vim which && \
```

File: container/Dockerfile_local [hunk 2]
```diff
@@ -22,15 +24,14 @@ USER faf
 ENV HOME /faf
 
 # Build faf
-RUN sed -i '/^pylint/s/$/ ||:/' faf.spec.in && \
-    ./autogen.sh && \
-    make rpm
+RUN sed -i '/^pylint/s/$/ ||:/' faf.spec && \
+    tito build --rpm --test
```
Comment (ernestask) [inline container/Dockerfile_local]:
```suggestion
RUN tito build --rpm --test
```

File: container/Dockerfile_local [hunk 3]
```diff
@@ -11,9 +13,9 @@ WORKDIR '/faf'
 # Change owner of /faf, clean git and install dependences
 RUN chown -R --silent faf /faf && \
     chmod -R --silent g=u /faf && \
-    dnf -y install rpm-build sudo git which vim make &&  \
+    dnf -y install git-core make rpm-build sudo tito vim which && \
```
Comment (mgrabovsky) [inline container/Dockerfile_local]: Do you think it would be worth it to move this installation above `COPY . /faf` so that it leverage the cache better?

Comment (mkutlak) [reply container/Dockerfile_local]: Ok, I did something. 

Please review.

**Pull request merged**